StripeIdentity/StripeIdentity/Source/API Bindings/Models/VerificationPage/VerificationPageStaticContentDocumentCaptureMBSettings.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  VerificationPageStaticContentDocumentCaptureMBSettings.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 1/24/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+
+extension StripeAPI {
+    /// Policy to configure capture strategy used to select the best frame.
+    enum MBCaptureStrategyType: String, Codable, Equatable, CaseIterable {
```

Comment:
What does MB stand for?

Reply:
MB stands for microblink(https://microblink.com/) we'll be integrating their [capture-ios](https://github.com/BlinkID/capture-ios) SDK